### DISCUSSION TITLE
[RAWeb API] How to get set type?

### DISCUSSION THREAD

**Discussion by WedgeSparda** [RetroAchievements/RAWeb#2637] [Ideas]
Hi, I want to update `API_GetGameList` to include the set type of the game. By this I mean the small badge next to the title indicating if it's a `Hack`,  `Prototype`, `Subset`, etc.

![Captura de pantalla 2024-08-23 a las 8 31 57](https://github.com/user-attachments/assets/ba5d3702-5d10-477b-9738-25af0388540b)

I checked the source code and found `tags` for games but I'm not sure if that's what I should be looking for. Also, don't know from which table in the database they're coming from.

Can someone point me in the right direction?

Thanks

**Comment by wescopeland** [comment]
Hi!

Right now these are unfortunately magic strings that prefix game titles. I’d prefer not to do this until we formally support tags.

**Comment by WedgeSparda** [comment]
Ok. Any estimate about when they'll be supported?

**Comment by WedgeSparda** [comment]
Oh, I see that titles come with `~Homebrew~` for example and is the frontend the one beautifying the title.
